api/config/default.json

```diff
@@ -18,7 +22,8 @@
     "database": "landgriffon",
     "synchronize": false,
     "migrationsRun": true,
-    "dropSchema": false
+    "dropSchema": false,
+    "logging": true
```

Comment:
So even tho I am now a proud #teamnative member, should we add an option into the dotenv skeleton file to make this option dynamic (without changing the code) for the poor people that haven't seen the light yet?

Reply:
https://github.com/typeorm/typeorm/blob/master/docs/logging.md

Typeorm expects a bool or an array here. I don't know how to put an array in a .env file, and I don't really feel like looking into that right now. If this is a concern, I can remove this change altogether, and we can tackle it later on, as part of a different task

Reply:
I understand. Not really a concern to me, it is clear enough how to change between logging levels.
Let's tackle this if someone else complains about the logging levels (which most likely won't happen)